mlflow/tensorflow.py

```diff
@@ -657,26 +657,67 @@ def autolog(every_n_iter=100):
                       "1.12 <= v <= 2.0.0 are supported.")
         return
 
+    @gorilla.patch(tensorflow.estimator.Estimator)
+    def train(self, *args, **kwargs):
+        global _AUTO_END_RUN
+        if not mlflow.active_run():
+            mlflow.start_run()
```

Comment:
As per comments on https://github.com/mlflow/mlflow/pull/2095, we should wrap this in a `try_mlflow_log` as well